src/main/resources/application.properties

```
@@ -10,3 +10,4 @@ spring.cloud.consul.discovery.register=true
 spring.cloud.consul.discovery.prefer-ip-address=true
 spring.cloud.consul.discovery.instance-id=${spring.application.name}:${spring.cloud.client.hostname}:${random.int[1,999999]}
 spring.cloud.consul.host=consul
+spring.rabbitmq.addresses=localhost:5672
```

```suggestion
spring.rabbitmq.host=rabbit
spring.rabbitmq.port=5672
```
The `application.properties` needs reference to the group's rabbit container instead of localhost because other services will not be able to reach localhost. Martin suggested that all services use the name **rabbit**. 

This can be tested by creating a local container named "rabbit" with the docker-compose file or by refering to localhost in one's `application-dev.properties`.

---

My bad, I forgot to change from localhost. I'll add the container in our docker-compose as well